tests/search_api/test_stats.py

```diff
@@ -6,6 +6,18 @@
 
 class TestStats(unittest.TestCase):
 
+    @mock.patch('pydoof.search_api.stats.SearchAPIClient')
+    def test_init_session(self, APIClientMock):
```

Comment:
I'm not sure how the tests works here, but... Should it be a tests for each flux? `Ok`, `Search engine not found`, `Bad request`,...

Reply:
Not really, this only tests that the client is called with the right parameters and URL. There is a different suit of tests that check that each Doophoenix response has the proper function return.